shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -469,9 +471,14 @@
         }
 
         item.setPosition(idx);
+        
```

Comment:
Looks like there's trailing spaces here - if you could get rid of those, us OCD folks would thank you :)